web-components/src/components/sliders/ResponsiveSlider.tsx

```diff
@@ -221,7 +229,7 @@ export default function ResponsiveSlider({
       <Grid
         container
         wrap="nowrap"
-        alignItems="center"
+        alignItems="flex-end"
```

Comment:
Could this change create visual regression with other used sliders or do you think it's safe to do?

Reply:
will set this as a prop instead to be sure